jazz_services/index.js

```diff
@@ -133,10 +133,6 @@ module.exports.handler = (event, context, cb) => {
                     logger.error('Error occured. ' + JSON.stringify(error, null, 2));
```

Comment:
The line `var data = result.fetchServices;` needs to be after if(error) 

Also `if(error) {`
should have a `return handleResponse(error, data ....);`

Reply:
changes are made as per the comment